{"uid":"d99a63c9d2eec702","name":"Проверка при нажатии в поп ап О системе открывается страница О системе","fullName":"tests.test_menu_profile.TestMenuProfile#test_pop_up_menu_click_button_about_system","historyId":"16d39dce0b4ab6569e65c78c76971951","time":{"start":1737931015870,"stop":1737931015870,"duration":0},"description":"Проверяется на пользователе Aleska","descriptionHtml":"<p>Проверяется на пользователе Aleska</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: Не найден элемент с локатором ('xpath', '//div[@class=\"onboarding_container__mDp2D\"]')","statusTrace":"request = <SubRequest 'login_to_app' for <Function test_pop_up_menu_click_button_about_system>>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"94a7b5cf6e7b89bdacf6c8595c9bbc1f\")>\n\n    @pytest.fixture(scope='function')\n    @allure.title('Авторизация пользователя в системе')\n    @allure.step('Авторизация пользователя в системе')\n    def login_to_app(request, driver):\n        \"\"\" Фикстура Логина в систему \"\"\"\n        login_page = ApiAuthorization(driver)\n        login_page.user(User.LOGIN, User.PASSWORD)\n        try:\n>           later = login_page.find_element(config.locators.RubricPageLocators.locators['Кнопка выберу потом'])\n\nconftest.py:55: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nbase/base_page.py:138: in find_element\n    return self.wait.until(EC.visibility_of_element_located(locator),\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"94a7b5cf6e7b89bdacf6c8595c9bbc1f\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x7f048414e340>\nmessage = 'Не найден элемент с локатором (\\'xpath\\', \\'//button[@data-testing=\"after_register_dialog__btn_choose_later\"]\\')'\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: Не найден элемент с локатором ('xpath', '//button[@data-testing=\"after_register_dialog__btn_choose_later\"]')\n\n../local/lib/python3.13/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nrequest = <SubRequest 'login_to_app' for <Function test_pop_up_menu_click_button_about_system>>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"94a7b5cf6e7b89bdacf6c8595c9bbc1f\")>\n\n    @pytest.fixture(scope='function')\n    @allure.title('Авторизация пользователя в системе')\n    @allure.step('Авторизация пользователя в системе')\n    def login_to_app(request, driver):\n        \"\"\" Фикстура Логина в систему \"\"\"\n        login_page = ApiAuthorization(driver)\n        login_page.user(User.LOGIN, User.PASSWORD)\n        try:\n            later = login_page.find_element(config.locators.RubricPageLocators.locators['Кнопка выберу потом'])\n            login_page.do_click(later)\n        except TimeoutException:\n>           element = login_page.find_element(\n                config.locators.MainPageLocators.locators['Листай вниз и найди свои интересы'])\n\nconftest.py:58: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nbase/base_page.py:138: in find_element\n    return self.wait.until(EC.visibility_of_element_located(locator),\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"94a7b5cf6e7b89bdacf6c8595c9bbc1f\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x7f048414dee0>\nmessage = 'Не найден элемент с локатором (\\'xpath\\', \\'//div[@class=\"onboarding_container__mDp2D\"]\\')'\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: Не найден элемент с локатором ('xpath', '//div[@class=\"onboarding_container__mDp2D\"]')\n\n../local/lib/python3.13/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1737925936720,"stop":1737925936721,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"driver","time":{"start":1737931015871,"stop":1737931027286,"duration":11415},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"setup","time":{"start":1737931027286,"stop":1737931027287,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Инициализация локаторов","time":{"start":1737931027287,"stop":1737931027287,"duration":0},"status":"passed","steps":[{"name":"Инициализация локаторов","time":{"start":1737931027287,"stop":1737931027287,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[{"name":"request","value":"<SubRequest 'elements' for <Function test_pop_up_menu_click_button_about_system>>"}],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":1,"hasContent":true,"attachmentStep":false},{"name":"Авторизация пользователя в системе","time":{"start":1737931027287,"stop":1737931188618,"duration":161331},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: Не найден элемент с локатором ('xpath', '//div[@class=\"onboarding_container__mDp2D\"]')\n\n","statusTrace":"  File \"/usr/local/lib/python3.13/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.13/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/usr/local/lib/python3.13/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/usr/local/lib/python3.13/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/usr/local/lib/python3.13/site-packages/_pytest/fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/usr/local/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/usr/workspace/conftest.py\", line 58, in login_to_app\n    element = login_page.find_element(\n              ~~~~~~~~~~~~~~~~~~~~~~~^\n        config.locators.MainPageLocators.locators['Листай вниз и найди свои интересы'])\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/workspace/base/base_page.py\", line 138, in find_element\n    return self.wait.until(EC.visibility_of_element_located(locator),\n           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n                           message=f'Не найден элемент с локатором {locator}')\n                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.13/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[{"name":"Авторизация пользователя в системе","time":{"start":1737931027287,"stop":1737931188616,"duration":161329},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: Не найден элемент с локатором ('xpath', '//div[@class=\"onboarding_container__mDp2D\"]')\n\n","statusTrace":"  File \"/usr/local/lib/python3.13/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"/usr/workspace/conftest.py\", line 58, in login_to_app\n    element = login_page.find_element(\n              ~~~~~~~~~~~~~~~~~~~~~~~^\n        config.locators.MainPageLocators.locators['Листай вниз и найди свои интересы'])\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/workspace/base/base_page.py\", line 138, in find_element\n    return self.wait.until(EC.visibility_of_element_located(locator),\n           ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n                           message=f'Не найден элемент с локатором {locator}')\n                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.13/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[{"name":"request","value":"<SubRequest 'login_to_app' for <Function test_pop_up_menu_click_button_about_system>>"},{"name":"driver","value":"<selenium.webdriver.chrome.webdriver.WebDriver (session=\"94a7b5cf6e7b89bdacf6c8595c9bbc1f\")>"}],"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":1,"hasContent":true,"attachmentStep":false}],"afterStages":[{"name":"driver::0","time":{"start":1737931189253,"stop":1737931199657,"duration":10404},"status":"passed","steps":[],"attachments":[{"uid":"863dba4437554a30","name":"page_source","source":"863dba4437554a30.html","type":"text/html","size":7893},{"uid":"babcf31fd39275aa","name":"screenshot","source":"babcf31fd39275aa.png","type":"image/png","size":52984},{"uid":"bac3641647743c84","name":"browser_logs","source":"bac3641647743c84.txt","type":"text/plain","size":9660}],"parameters":[],"shouldDisplayMessage":false,"attachmentsCount":3,"stepsCount":0,"hasContent":true,"attachmentStep":false}],"labels":[{"name":"severity","value":"Critical"},{"name":"feature","value":"Меню профиля"},{"name":"tag","value":"regression"},{"name":"tag","value":"menu_profile"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_menu_profile"},{"name":"subSuite","value":"TestMenuProfile"},{"name":"host","value":"69e45c3075d5"},{"name":"thread","value":"1-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_menu_profile"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"critical","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":2,"broken":3,"skipped":0,"passed":31,"unknown":0,"total":36},"items":[{"uid":"2d5201f5611c0641","status":"passed","time":{"start":1737843163387,"stop":1737843168129,"duration":4742}},{"uid":"7b35912377e43e29","status":"passed","time":{"start":1737832364922,"stop":1737832369156,"duration":4234}},{"uid":"9b8e9222a11dab41","status":"passed","time":{"start":1737799724872,"stop":1737799729123,"duration":4251}},{"uid":"44acef73ffea87fd","status":"passed","time":{"start":1737757953637,"stop":1737757957883,"duration":4246}},{"uid":"c1c3b31021e23f81","status":"passed","time":{"start":1737744954378,"stop":1737744958105,"duration":3727}},{"uid":"d506e20e804678d1","status":"failed","statusDetails":"AssertionError: Ожидался текст: 'О системе', отображается текст: 'Aleska'","time":{"start":1737652806602,"stop":1737652809319,"duration":2717}},{"uid":"156d36e6ecced11d","status":"failed","statusDetails":"AssertionError: Ожидался текст: 'О системе', отображается текст: 'Aleska'","time":{"start":1737570515304,"stop":1737570517028,"duration":1724}},{"uid":"53d7e17bf676bf8b","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/chromium-browser is no longer running, so ChromeDriver is assuming that Chrome has crashed.)","time":{"start":1737568966350,"stop":1737568966350,"duration":0}},{"uid":"17ccd71879968e11","status":"passed","time":{"start":1734034780380,"stop":1734034785148,"duration":4768}},{"uid":"c7b3342f9b006b8","status":"passed","time":{"start":1733937464490,"stop":1733937467179,"duration":2689}},{"uid":"fdd2d3b67be14d97","status":"passed","time":{"start":1733866065418,"stop":1733866068466,"duration":3048}},{"uid":"8d8770def781e756","status":"passed","time":{"start":1733860461050,"stop":1733860463811,"duration":2761}},{"uid":"186ccc4db2ac483c","status":"passed","time":{"start":1733857041825,"stop":1733857044028,"duration":2203}},{"uid":"b30309f4275c6fb5","status":"passed","time":{"start":1733771925782,"stop":1733771930351,"duration":4569}},{"uid":"8426e52f4dc42841","status":"passed","time":{"start":1733766379912,"stop":1733766382084,"duration":2172}},{"uid":"a05deefa6224cf0b","status":"passed","time":{"start":1733421941950,"stop":1733421944648,"duration":2698}},{"uid":"1364595f6e242bf8","status":"passed","time":{"start":1726780401555,"stop":1726780404252,"duration":2697}},{"uid":"e2b5c9ee7b064f30","status":"passed","time":{"start":1726777125872,"stop":1726777129079,"duration":3207}},{"uid":"dbcf51c38bd04077","status":"passed","time":{"start":1724098904333,"stop":1724098907022,"duration":2689}},{"uid":"52613016504d44d","status":"passed","time":{"start":1723829151568,"stop":1723829154742,"duration":3174}}]},"tags":["regression","menu_profile"]},"source":"d99a63c9d2eec702.json","parameterValues":[]}